src/telemetry/server/ceph_telemetry/rest/index.py

```diff
@@ -0,0 +1,7 @@
+from flask import jsonify
+from flask_restful import Resource
+
+
+class Index(Resource):
+    def get(self):
+        return jsonify(status=True)
```

Comment:
i don't really understand how this helps us to access the collected stats?

Reply:
It is a thing of me to make sure it responds on a GET so that you can do a health check with a external service to see if the webservice is alive.